### ISSUE TITLE
Windows javafx

### ISSUE THREAD

**Issue by solidunity2021** [microsoft/java-wdb#35]
Please make javafx run on windows os by default
So we can create applications and run it easily on the system and share it with others windows os system

**Comment by brunoborges** [comment]
JavaFX combined with `jpackage` gives developers a solution to produce MSI installers that bundle the JVM.

We suggest going through that route.

**Comment by credmond** [comment]
Yeah. Just make sure your users don't mind waiting 20+ seconds for your application to start, while Windows Defender scans a bunch of scary bytecode. Over and over.
